Core/Object Arts/Dolphin/Base/UtfEncodedString.cls

```diff
@@ -61,6 +61,18 @@ copyWith: newElement
 
 	^self, (self class with: newElement)!
 
+copyWithout: aCharacter
+	"Answer a <String> which is a copy of the receiver, but in
+	which all occurrences of the <Character> aCharacter have been removed.
+
+	N.B. Use a ReadStream to process whole UTF-8 codepoints at once--otherwise
+	we will be unable to recognize non-ASCII characters in a UTF-8 string."
+
+	| aStream |
+	aStream := (self copyLikeOfSize: self copySize) writeStream.
```

Comment:
This doesn't need to be so generic at this point in the hierarchy.  It could just be the simpler `self class writeStream: self size` here. But it doesn't matter, I'll change that at some later point.